nni/common/device.py

```diff
@@ -0,0 +1,28 @@
+
+from dataclasses import dataclass
+
+@dataclass
+class GPUDevice:
+    node_id: str
+    gpu_id: int
+    status: str = 'free'
+
+    def __eq__(self, o) -> bool:
+        return self.host == o.host and self.gpu_id == o.gpu_id
```

Comment:
where is the definition of `host`?

Reply:
Updated. There should be only `host`. No `node_id`.